app/models/user.rb

```diff
@@ -6,20 +6,20 @@ class User < ActiveRecord::Base
   validates :email, presence: true,
                     length: { in: 5..50 },
                     uniqueness: true
-  validates :display_name, allow_blank: true, 
+  validates :display_name, allow_blank: true,
                             length: {in: 2..32}
 
   has_secure_password
   enum role: %w(default admin)
-  
+
   has_attached_file :avatar, styles: {thumb: '100x100>',
                                       square: '200x200#',
-                                      medium: '300x300>'}, 
+                                      medium: '300x300>'},
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.